admin_guide/upgrades.adoc

```diff
@@ -162,13 +162,13 @@ routing service].
 ====
 If you are upgrading to
 ifdef::openshift-origin[]
-OpenShift Origin 1.0.4 or later, first see the
+OpenShift Origin 1.0.4, first see the
 link:#additional-instructions-per-release[Additional Instructions per Release]
 section for important steps link:#openshift-origin-1-0-4[specific to this
 upgrade],
 endif::[]
 ifdef::openshift-enterprise[]
-OpenShift Enterprise 3.0.1.0 or later, first see the
+OpenShift Enterprise 3.0.1.0, first see the
```

Comment:
Wasn't sure initially whether to put "or later" here or not... Deciding in hindsight against it.